CorsixTH/Lua/dialogs/resizables/main_menu.lua

```diff
@@ -111,7 +111,7 @@ function UIMainMenu:draw(canvas, x, y)
   -- The following strings are drawn in reverse order
   local ly = y + self.height - 15
   -- Move the version string up a bit if showing check for updates disabled warning.
-  if not TheApp.config.check_for_updates then
+  if TheApp:isUpdateCheckDisabledByConfig() then
```

Comment:
Good catch but my concern was also here
![image](https://github.com/CorsixTH/CorsixTH/assets/20030128/99c1046f-3acb-447b-9731-a7e66787e71e)

Reply:
Ah... I'll take care of tgat one too.